src/DynamoCore/Configuration/IPathResolver.cs

```diff
@@ -158,6 +158,12 @@ public interface IPathManager
         string PreferenceFilePath { get; }
 
         /// <summary>
+        /// Full path to the python template py file. This setting file is specific 
+        /// to the current user.
+        /// </summary>
+        string PythonTemplateFilePath { get; }
```

Comment:
Thanks @mjkkirschner , was definitely not aware of Dynamo Revit was implementing its own class from this interface. Will try and keep an eye out for that in the future.